styles/index.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  text-align: center;
+  font-size: 18px;
+}
+body {
+  background-color: #e8f3d6;
+  text-align: center;
+  color: #545454;
+}
+button {
+  background-color: #8caf9a;
+  border: none;
+  color: white;
+  padding-left: 2em;
+  padding-right: 2em;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 10px 10px;
+}
+h1 {
+  font-size: 32px;
+  display: inline;
+  margin: 20px;
+  font-size: 3.5rem;
+  font-family: 'Sacramento', cursive;
+  color: #8B7E74;
+}
+h2 {
+  border: 30px;
+  color: #545454;
+  display: inline;
+}
+.container {
+  display: grid;
+  text-align: center;
+  grid-template-rows: 25% 30% 20% 25%;
+  grid-template-columns: 30% 70%;
+  margin: auto;
+  width: auto;
+  height: 800px;
+}
+.header {
+  grid-row: 1/2;
+  grid-column: 1/3;
+  margin: 20px 2em;
+  vertical-align: middle;
+}
```

Comment:
Try to avoid using class names that are the same name as an HTML element (like `header`).